keps/sig-node/3063-dynamic-resource-allocation/README.md

```diff
@@ -2729,7 +2730,65 @@ already received all the relevant updates (Pod, ResourceClaim, etc.).
 
 ###### What are other known failure modes?
 
-To be added for beta.
+- DRA driver does not or cannot allocate a resource claim.
+
+  - Detection: The primary mechanism is through vendors-provided monitoring for
+    their driver. That monitor needs to include health of the driver, availability
+    of the underlying resource, etc. The common helper code for DRA drivers
+    posts events for a ResourceClaim when an allocation attempt fails.
+
+    When pods fail to get scheduled, kube-scheduler reports that through events
+    and pod status. For DRA, that includes "waiting for resource driver to
+    provide information" (node not selected yet) and "waiting for resource
```

Comment:
Do these events/status updates include information about the specific claim/driver that is blocking progress?

Reply:
When waiting for a claim, the claim is mentioned. This allows the user to drill down and check the claim.

When waiting for PodSchedulingContext information, that object is not mentioned explicitly. It doesn't need to be named because the name is the same as for the pod.

In both cases it is assumed that users understand the concepts enough to know what "claim" and "pod scheduling context" are.